ReadMe.md

```diff
@@ -22,6 +22,8 @@ Run the following command to clone WebKit's Git repository:
 git clone https://github.com/WebKit/WebKit.git WebKit
 ```
 
+Using [git fsmonitor](https://git-scm.com/docs/git-config#Documentation/git-config.txt-corefsmonitor) will make many git commands faster (such as `git status`).
```

Comment:
Probably would at least make sense to actually say what commands to run..

Reply:
Good point, I've added the actual config line. WDYT ?